src/main/webapp/webpack.config.js

```diff
@@ -97,6 +101,17 @@ module.exports = (env, argv) => {
               new CssMinimizerPlugin({ parallel: true }),
               new TerserPlugin({ parallel: true, include: /\/resources/ }),
             ],
+            runtimeChunk: "single",
+            splitChunks: {
+              name: false,
+              chunks(chunk) {
+                // exclude modals in projects-samples-*
+                return (
```

Comment:
Is this just removing the project samples modals from the runtime chunk?

Reply:
Correct